pkg/postage/stamper_test.go

```diff
@@ -19,29 +19,13 @@ import (
 
 // TestStamperStamping tests if the stamp created by the stamper is valid.
 func TestStamperStamping(t *testing.T) {
-	privKey, err := crypto.GenerateSecp256k1Key()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	owner, err := crypto.NewEthereumAddress(privKey.PublicKey)
-	if err != nil {
-		t.Fatal(err)
-	}
-	signer := crypto.NewDefaultSigner(privKey)
-	createStamp := func(t *testing.T, stamper postage.Stamper) (swarm.Address, *postage.Stamp) {
-		t.Helper()
-
-		chunkAddr := swarm.RandAddress(t)
-		stamp, err := stamper.Stamp(chunkAddr)
-		if err != nil {
-			t.Fatal(err)
-		}
-		return chunkAddr, stamp
-	}
+	t.Parallel()
 
 	// tests a valid stamp
 	t.Run("valid stamp", func(t *testing.T) {
+		t.Parallel()
+
+		owner, signer := generateOwnerAndSigner(t)
```

Comment:
Do we have to generate/get `owner, signer` every time for each test?

Reply:
If we generate/get `owner, signer` just once  as we were doing before, then the test fails with the following error
```
ethersphere/bee/v2/pkg/postage

--- FAIL: TestStamperStamping (0.01s)
    --- FAIL: TestStamperStamping/reuse_index_but_get_new_timestamp_for_mutable_or_immutable_batch (0.00s)
        /Users/ersonpereira/Workspace/work/bee/pkg/postage/stamper_test.go:142: expected no error, got owner mismatch
    --- FAIL: TestStamperStamping/valid_stamp (0.00s)
        /Users/ersonpereira/Workspace/work/bee/pkg/postage/stamper_test.go:53: expected no error, got owner mismatch
FAIL
FAIL	github.com/ethersphere/bee/v2/pkg/postage	0.513s
FAIL
```

So generating `owner, signer` for each test fixes this issue.